specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -3381,8 +3381,8 @@
           "$ref": "../datafactory.json#/definitions/SecretBase"
         },
         "clientId": {
-          "description": "The client id of the google application used to acquire the refresh token.",
-          "$ref": "../datafactory.json#/definitions/SecretBase"
+          "description": "The client id of the google application used to acquire the refresh token. Type: string (or Expression with resultType string).",
+          "type": "object"
```

Comment:
Do we need a `properties` field here?

Reply:
Yes, the property is already here, I just modify the property type from SecretBase to object. It is an issue with SecretBase.